templates/commands/goldentest/verify.go

```diff
@@ -130,7 +130,8 @@ func (c *VerifyCommand) Run(ctx context.Context, args []string) error {
 			goldenContent, err := os.ReadFile(goldenFile)
 			if err != nil {
 				if errors.Is(err, os.ErrNotExist) {
-					failureText := red(fmt.Sprintf("-- [%s] generated, however not recorded in test data", goldenFile))
+					failureText := red(fmt.Sprintf("-- [%s] generated, however not recorded in test data, you might "+
```

Comment:
It's probably more user-friendly to print this once per failing test, rather than once per file.

Reply:
updated. I still keep this failureText to let users know which file is missing recorded test data in case they don't use record subcommand and write the test data by themselves.